.travis.yml

```diff
@@ -27,8 +27,9 @@ install:
   - conda config --add channels conda-forge
   - conda update -q conda
   - conda info -a
-  - conda create -n test-environment python=$TRAVIS_PYTHON_VERSION pyflakes pytest pytest-doctestplus numpy sympy hypothesis doctr sphinx myst-parser sphinx_rtd_theme pytest-cov pytest-flakes
```

Comment:
It seems as though there are still some numpy imports

Reply:
NumPy is still a dependency. I'm just installing it from git now.